extensions/data-transfer/portability-data-transfer-instagram/src/main/java/org/dataportabilityproject/transfer/instagram/photos/InstagramPhotoExporter.java

```diff
@@ -66,7 +63,7 @@ public InstagramPhotoExporter(ObjectMapper objectMapper, HttpTransport httpTrans
   @Override
   public ExportResult<PhotosContainerResource> export(UUID jobId, TokensAndUrlAuthData authData,
       ExportInformation exportInformation) {
-    return exportPhotos(authData, Optional.of(exportInformation.getPaginationData()));
+    return exportPhotos(authData, Optional.ofNullable(exportInformation.getPaginationData()));
```

Comment:
Good catch! We should check the other Exporters when you get a chance.

Reply:
Thanks! It threw a NPE when testing. I'll do a pass through for the others.